library/general/src/lib/ui/greasemonkey.rb

```diff
@@ -86,6 +86,12 @@ def VStackFrames(old)
     end
     module_function :VStackFrames
 
+    # @param old [Yast::Term]
+    # @return    [Yast::Term]
+    # @example
+    #   term(:FrameWithMarginBox, "Title", "arg1", "arg2")
+    #      ->
+    #   Frame("Title", MarginBox(1.45, 0.45, "arg1", "arg2"))
```

Comment:
seen this usage, I think it make sense to mention it as old example and deprecated. I think prefered way to use should be

```
include GreaseMonkey

def contents
  VBox(
    FrameWithMarginBox("Title", "arg1", "arg2")
  end
end
```

and this should be a priority example for suggested usage. Or how do you see it?